src/util/rebind.js

```diff
@@ -34,6 +34,8 @@ function capitalizeFirstLetter(str) {
  * be rebound.
  */
 export function rebindAll(target, source, prefix, exclusions) {
+    prefix = typeof prefix !== 'undefined' ? prefix : '';
```

Comment:
This doesn't look right - I don't think I have touched rebind. Might be a bad merge.

I'll look into this.